.github/CHANGELOG.md

```diff
@@ -2,6 +2,44 @@
 
 <h3>New features since last release</h3>
 
+* The new Device Tracker capabilities allows for flexible and versatile tracking of executions, even inside parameter-shift gradients. This functionality will improve the ease of monitoring large batches and remote jobs.
+[(#1355)](https://github.com/PennyLaneAI/pennylane/pull/1355)
+
+```python
+dev = qml.device('default.qubit', wires=1, shots=100)
+
+@qml.qnode(dev, diff_method="parameter-shift")
+def circuit(x):
+    qml.RX(x, wires=0)
+    return qml.expval(qml.PauliZ(0))
+
+x = np.array(0.1)
+
+with qml.Tracker(circuit.device) as tracker:
```

Comment:
Why not pass `dev` to `qml.Tracker` here?

Reply:
When creating a QNode with backpropogation, the device you create it with is not the device where the execution happens. Though this is a parameter-shift qnode, I want to familiarize people with calling ``circuit.device`` instead of ``dev``, so we don't get problems in backpropogation mode.

Reply:
This is a subtlety that is worth considering deeper & documenting in more details. The fact that the device is being swapped under the hood is not user facing at the moment.